azure-wildfly/src/test/java/sunstone/azure/armTemplates/di/AzStandaloneManagementClientTests.java

```diff
@@ -22,19 +22,22 @@
         @Parameter(k = "virtualMachineName", v = AzureTestConstants.instanceName),
         @Parameter(k = "imageRefId", v = AzureTestConstants.IMAGE_REF)
 },
-        template = "sunstone/azure/armTemplates/eap.json", group = SunstoneResourceAzSuiteTests.group, perSuite = true)
+        template = "sunstone/azure/armTemplates/eap.json", group = AzStandaloneManagementClientTests.group, perSuite = true)
 public class AzStandaloneManagementClientTests {
+    public static final String group = "${azure.group:sunstone-testing-group}";
 
     @AzureVirtualMachine(name = AzureTestConstants.instanceName)
     static OnlineManagementClient staticMngmtClient;
 
-    @AzureVirtualMachine(name = AzureTestConstants.instanceName, group = SunstoneResourceAzSuiteTests.group, mode = OperatingMode.STANDALONE, standalone = @StandaloneMode(user = AzureTestConstants.mngmtUser, password = AzureTestConstants.mngmtPassword, port = AzureTestConstants.mngmtPort))
+    @AzureVirtualMachine(name = AzureTestConstants.instanceName, group = AzStandaloneManagementClientTests.group)
```

Comment:
One can even say, having it separated like this is even cleaner/nicer. Btw. haven't we discussed this option in past?

But we can see (in this MR) it comes with bigger complexity/work. Now question is if it is worth in long term.

Reply:
We might have:)

Yeah, te project is more complex. Modules kinda extending other modules and depending on them, not really something that is widely used. However, the vision is to enable cross team collaboration. How much does it worth it is yet to be determined.